Automattic/wp-calypso
# Plans step redesign: Implement the header section of each plan

#### Changes proposed in this Pull Request

This PR implements the experiment that redesigns the signup flow plans step. Users in the treatment group will see the new header section for each plan in the signup plans step.

NOTE: 
1. This PR uses the changes in https://github.com/Automattic/wp-calypso/pull/49877 as the base branch and is built on top of it.
2. This PR only implements the header section. The rest of the plan sections are implemented in subsequent PRs.

**Important links**
1. [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1362%3A813)
2. pau2Xa-2yE-p2 for the project thread.


**Annual plans header:**

<img width="1655" alt="Screenshot 2021-02-19 at 11 01 54 AM" src="https://user-images.githubusercontent.com/1269602/108462060-f293b680-72a1-11eb-91d4-95118acfe83d.png">


**Monthly plans header:**

<img width="1658" alt="Screenshot 2021-02-19 at 11 02 02 AM" src="https://user-images.githubusercontent.com/1269602/108462079-f9222e00-72a1-11eb-98ad-85c9bf5e780d.png">

#### Testing instructions



* Open http://calypso.localhost:3000/ and assign yourself to treatment of the `signup_plans_step_redesign_v1` experiment in the Abacus tool.
* Begin signup at `/start`and verify that the plans step UI matches the [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1362%3A813). Verify for both free subdomain and custom domain selections.
* Verify that the UI matches the Figma design for both annual and monthly plan views.
* Confirm that in Calypso's /plans view, the UI is unchanged. 
* Also confirm that the launch flow UI is unchanged.
* Verify that `control` experience is unchanged. 



Related to https://github.com/Automattic/wp-calypso/pull/49877
